components/filter/test/filter-tags.visual-diff.js

```diff
@@ -110,7 +110,7 @@ describe('d2l-filter-tags', () => {
 			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { captureBeyondViewport: false, clip: rect });
 		});
 
-		it('is correct after clicking Clear All with label', async function() {
+		it('is correct after clicking Clear All with two filters', async function() {
```

Comment:
Even with the label gone, I think this test is a good one to have to make sure both filters were cleared.